### PR TITLE
Add Features section to the readme

### PR DESCRIPTION
**Pull request by yurishkuro**
Signed-off-by: Yuri Shkuro <ys@uber.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,52 @@ See also:
...
+## Features
+
+### High Scalability
+
+Jaeger backend is designed to have no single points of failure and to scale with the business needs.
+For example, any given Jaeger installation at Uber is typically processing several billions of spans per day.
+
+### Native support for OpenTracing
+
+Jaeger backend, Web UI, and instrumentation libraries have been designed from grounds up to support the OpenTracing standard.
```
Comment (black-adder) [inline README.md]: "from the ground up" I think is the correct phrase

File: README.md [hunk 2]
```diff
@@ -24,6 +24,52 @@ See also:
 
 Jaeger is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CNCF). If you are a company that wants to help shape the evolution of technologies that are container-packaged, dynamically-scheduled and microservices-oriented, consider joining the CNCF. For details about who's involved and how Jaeger plays a role, read the CNCF [announcement](https://www.cncf.io/blog/2017/09/13/cncf-hosts-jaeger/).
 
+## Features
+
+### High Scalability
+
+Jaeger backend is designed to have no single points of failure and to scale with the business needs.
+For example, any given Jaeger installation at Uber is typically processing several billions of spans per day.
```
Comment (black-adder) [inline README.md]: Do you wanna link to http://jaeger.readthedocs.io/en/latest/deployment/#collectors or describe how it's scalable somewhere in the docs?

File: README.md [hunk 3]
```diff
@@ -24,6 +24,52 @@ See also:
...
+
+### Native support for OpenTracing
+
+Jaeger backend, Web UI, and instrumentation libraries have been designed from grounds up to support the OpenTracing standard.
+  * Representing traces as directed acyclic graphs (not just trees) via [span references](https://github.com/opentracing/specification/blob/master/specification.md#references-between-spans)
```
Comment (black-adder) [inline README.md]: these three bullet points will read better if they all start in the same way, ie "Represent, support, support"

File: README.md [hunk 4]
```diff
@@ -24,6 +24,52 @@ See also:
...
+
+Jaeger supports two popular open source NoSQL databases as trace storage backends: Cassandra 3.4+ and Elasticsearch 5.x/6.x.
+The are ongoing community experiments using other databases, such as ScyllaDB, InfluxDB, Amazon DynamoDB. Jaeger also ships
+with a simple in-memory storage for testing setups.
+
+### Modern Web UI
+
+Jaeger Web UI is implemented in Javascript using popular open source frameworks like React. A number of performance
+improvements have been released in v1.0 to allow the UI to efficiently deal with large volumes of data, and to display
+traces with up to 50,000 spans.
```
Comment (black-adder) [inline README.md]: ive used it to display 80,000 spans. Maybe rephrase as: "tens of thousands of spans" here instead? I'd rather not set a hard upper bound in the docs.

File: README.md (hunk 2 above)
Comment (yurishkuro) [reply README.md]: we can change it later once we have the docs. The link to collectors seems a bit arbitrary, as the no-SPOF argument must apply to the whole stack.

**Comment by yurishkuro** [comment]
@black-adder updated, give it another read.

File: README.md [hunk 5]
```diff
@@ -24,6 +24,52 @@ See also:
...
+
+Jaeger backend, Web UI, and instrumentation libraries have been designed from ground up to support the OpenTracing standard.
+  * Represent traces as directed acyclic graphs (not just trees) via [span references](https://github.com/opentracing/specification/blob/master/specification.md#references-between-spans)
+  * Support strongly typed span _tags_ and _structured logs_
+  * Support general distributed context propagation mechanism via _baggage_
+
+### Multiple storage backends
+
+Jaeger supports two popular open source NoSQL databases as trace storage backends: Cassandra 3.4+ and Elasticsearch 5.x/6.x.
+The are ongoing community experiments using other databases, such as ScyllaDB, InfluxDB, Amazon DynamoDB. Jaeger also ships
```
Comment (black-adder) [inline README.md]: s/The/There

File: README.md [hunk 6]
```diff
@@ -24,6 +24,52 @@ See also:
...
+### Cloud Native Deployment
+
+Jaeger backend is distributed as a collection of Docker images. The binaries support various configuration methods,
+including command line options, environment variables, and configuration files in multiple formats (yaml, toml, etc.)
+Deployment to Kubernetes clusters is assisted by [Kubernetes templates](https://github.com/jaegertracing/jaeger-kubernetes)
+and a [Helm chart](https://github.com/kubernetes/charts/tree/master/incubator/jaeger).
+
+### Observability
+
+All Jaeger backend components expose [Prometheus](https://prometheus.io/) metrics by default (other metrics backends are
```
Comment (black-adder) [inline README.md]: that reminds me, we should have some guide on explaining how users can set this up.

Comment (yurishkuro) [reply README.md]: someone booked this https://github.com/jaegertracing/jaeger/issues/611

Comment (black-adder) [reply README.md]: ahh, what a go-getter that person is.
